start.R

```diff
@@ -327,7 +335,7 @@ if (any(c("--reprepare", "--restart") %in% argv)) {
     } else {
       if (! is.null(cfg[["backup"]][["cm_nash_mode"]])) cfg$gms$cm_nash_mode <- cfg$backup$cm_nash_mode
     }
-    if ("--testOneRegi" %in% argv) {
+    if (any(c("--quickRun", "--testOneRegi") %in% argv)) {
       if (is.null(cfg[["backup"]][["optimization"]])) cfg$backup$optimization <- cfg$gms$optimization
```

Comment:
I would add `cfg$gms$cm_iteration_max <- 1` but only for quickRun to make sure it is always quick independent of the settings in default.cfg or scenario config file.